memo-app/src/hooks/useMemos.jsx

```diff
@@ -0,0 +1,38 @@
+import useLocalStorage from "./useLocalStorage.jsx";
+
+export default function useMemos(isAdding, selectedId) {
+  const [memos, setMemos] = useLocalStorage("memo");
+  const selectedMemo = memos.find((memo) => memo.id === selectedId);
+
+  function saveMemo(content) {
+    const newId = crypto.randomUUID();
+
+    if (isAdding) {
+      setMemos([
+        ...memos,
+        {
+          id: newId,
```

Comment:
nits

```suggestion
          id: crypto.randomUUID(),
```

で良さそう！ randomUUID 使うの良い 👍

Reply:
修正しました！